std_msgs/Float32 heading
std_msgs/Float32 distance
std_msgs/Bool return_ground_truth
---
gazebo_msgs/ModelState ground_truth
std_msgs/Float32 noisy_heading
std_msgs/Float32 noisy_distance
turtlebot_ctrl/TurtleBotScan scan_data